jdaviz/core/marks.py

```diff
@@ -0,0 +1,104 @@
+from bqplot.marks import Lines
+from specutils.spectra.spectrum1d import Spectrum1D
+import astropy.units as u
+
+from glue.core import HubListener
```

Comment:
Total nitpick on my part. 😸 
```suggestion
from astropy import units as u
from bqplot.marks import Lines
from glue.core import HubListener
from specutils import Spectrum1D
```